conftest.py

```diff
@@ -1,19 +1,49 @@
 
 import pytest
+import logzero
+import configparser
+import os
 
 
 def pytest_addoption(parser):
-    parser.addoption("--type", action="store", default="aks", help="PCP selection: aks/eks")
-    parser.addoption("--nocleanup", action="store_true", default=False, help="Disable cleanup")
-    parser.addoption("--imageid", action="store", default="", help="Image id name")
-    parser.addoption("--bloburi", action="store", default="", help="URL to image blob")
+    parser.addoption("--type", action="store", default="aks",
+                     help="PCP selection: aks/eks")
+    parser.addoption("--nocleanup", action="store_true",
+                     default=False, help="Disable cleanup")
+    parser.addoption("--skipterraform", action="store_true",
+                     default=False, help="Not invoke terraform at all")
+    parser.addoption("--imageid", action="store",
+                     default="", help="Image id name")
+    parser.addoption("--bloburi", action="store",
+                     default="", help="URL to image blob")
```

Comment:
Not very important, but wouldn´t it be more readable using underscores? `image_id` `no_cleanup` `skip_terraform` `blob_uri`

Reply:
ok